pallets/loans/docs/types.md

```diff
@@ -156,7 +156,7 @@ package pricing {
             price_id: PriceId,
             max_borrow_quantity: MaxBorrowAmount,
             notional: Balance,
-            pool_id: PoolId
+            max_variation_price: Perthing,
```

Comment:
I think this name is worse than `max_price_delta`: variation technically is a change/delta over time (as in, the variance of a series of data points, rather than the difference of one data point to another)